pkg/utils/installer/install.go

```diff
@@ -25,11 +25,14 @@ import (
 
 	"github.com/minio/direct-csi/pkg/utils"
 
+	admissionv1 "k8s.io/api/admissionregistration/v1"
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
 	storagev1 "k8s.io/api/storage/v1"
 	storagev1beta1 "k8s.io/api/storage/v1beta1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	kErr "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
This is a nit - can we make this `kerr` instead of `kErr`. I'll merge this PR, but as a part of the next PR, we should take care of this